sdk/core/azure-core/inc/azure/core/uuid.hpp

```diff
@@ -39,10 +40,23 @@ namespace Azure { namespace Core {
      */
     std::string ToString();
 
+    /**
+     * @brief Returns the binary value of the Uuid for consumption by clients who need non-string
+     * representation of the Uuid
+     * @returns An array with the binary representation of the Uuid.
+     */
+    std::array<uint8_t, UuidSize> const& GetAsArray() const { return m_uuid; }
```

Comment:
I think `AsArray()` name is better than `GetAsArray()`

Reply:
My preference as well, but @ahsonkhan felt GetAsArray was a better name.

Reply:
Either is good. `AsArray` works well.